src/analysis/asymptotics.lean

```diff
@@ -383,12 +403,12 @@ by simp only [is_O_with, norm_norm]
 alias is_O_with_norm_right ↔ asymptotics.is_O_with.of_norm_right asymptotics.is_O_with.norm_right
 
 @[simp] theorem is_O_norm_right : is_O f (λ x, ∥g' x∥) l ↔ is_O f g' l :=
-exists_congr $ λ _,  is_O_with_norm_right
+by unfold is_O; exact exists_congr (λ _, is_O_with_norm_right)
```

Comment:
```suggestion
by { unfold is_O, exact exists_congr (λ _, is_O_with_norm_right) }
```
Here and below, can you use the `by { t1, t2 }` idiom instead of `by t1; t2`?